### PR TITLE
Add umbrella chart and update Helm releaser

### PR DESCRIPTION
**Pull request by cvlc**
Tested on EKS.

### CODE REVIEW

File: charts/ondat/values.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+ondat-operator:
```
Comment (aeroniero33) [inline charts/ondat/values.yaml:1]: We should probably add all the possible values here. https://github.com/ondat/charts/blob/umbrella_chart/charts/ondat-operator/values.yaml

Comment (cvlc) [reply charts/ondat/values.yaml:1]: What do you think about having links to the values.yaml for the respective charts in comments in this values.yaml file? That way, it's fairly easy to know where to go for a full reference!

File: charts/ondat/README.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+# Install the chart in a namespace
+$ kubectl create namespace ondat
+$ helm install ondat ondat/ondat \
+    --namespace ondat
+```
+
+Or, alternatively, with a custom `values.yaml` file:
+
+```console
+$ helm install ondat/etcd-cluster-operator \
```
Comment (DavidMarchant) [inline charts/ondat/README.md:44]: I think this should be ondat/ondat

**Review by DavidMarchant — CHANGES_REQUESTED** [review]
Still haven't been able to test this yet without access to the remote but it looks good to me!

One tiny change but happy with it if it's been tested

The only other question is the name, i think just ondat is fine but might be a little confusing versus ondat-operator, maybe ondat-stack? That's the terminology prometheus uses. Ultimately happy with either though!

File: charts/ondat/README.md (hunk 2 above)
Comment (cvlc) [reply charts/ondat/README.md:44]:
```suggestion
$ helm install ondat/ondat \
```
Yes, you're right! Great spot 🙏

**Comment by cvlc** [comment]
I think the reason I like ondat is that it's very simple - my hope is that it's obviously 'the one' to go for, for new users, but I'm totally open to ondat-stack - does anybody else feel strongly either way?

File: charts/ondat/values.yaml (hunk 1 above)
Comment (aeroniero33) [reply charts/ondat/values.yaml:1]: I think having a link to the rest of the values would be a good middleground 🙂

**Review by DavidMarchant — APPROVED** [review]
Just `ondat` is good with me! Thanks Calum

**Pull request merged**